rfcs/20200712-tfrt-kernel-fallback.md

```diff
@@ -0,0 +1,641 @@
+# Title of RFC
+
+| Status        | (Proposed / Accepted / Implemented / Obsolete)          |
+| :------------ | :------------------------------------------------------ |
+| **RFC #**     | [NNN](https://github.com/tensorflow/community/pull/NNN) |
+| **Author(s)** | Anna Revinskaya (annarev@google.com), Jeremy Lau        |
+:               : (lauj@google.com)                                       :
+| **Sponsor**   | Jeremy Lau (lauj@google.com)                            |
+| **Updated**   | 2020-07-06                                              |
+
+## Objective
+
+This proposal focuses on getting majority of “well-behaved” ops running in
+[TF Lite](https://www.tensorflow.org/lite) by skipping current eager runtime and
+calling kernels directly in [TFRT](https://github.com/tensorflow/runtime) (a new
```

Comment:
I would suggest a diagram that shows what this change really looks like in system architecture. It will also elucidate "directly" means. Show a box for the eager api, the opkernelcontext, etc.

Reply:
Good suggestion to add diagrams. I minimal side-by-side diagrams for Kernel Fallback and Runtime Fallback.
I also included a more detailed diagram under Design Proposal.